azurerm/internal/services/eventgrid/tests/eventgrid_domain_topic_resource_test.go

```diff
@@ -0,0 +1,150 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMEventGridDomainTopic_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_eventgrid_domain_topic", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMEventGridDomainTopicDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMEventGridDomainTopic_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMEventGridDomainTopicExists(data.ResourceName),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "id"),
```

Comment:
thinking about it we can remove this - since we'll have failed to find the resource (above) and the import test (below) if this isn't set

```suggestion
```